org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/GuardedPattern.java

```diff
@@ -259,9 +261,9 @@ public void setExpression(Expression expression) {
 
 	/**
 	 * Sets the pattern of this switch case.
-	 * @noreference This method is not intended to be referenced by clients.
-	 * @exception UnsupportedOperationException if this operation is used not for JLS18
+	 * @exception UnsupportedOperationException if this operation is used not for JLS21
 	 * @exception UnsupportedOperationException if this operation is used without previewEnabled
```

Comment:
this comment to be removed - now its a standard feature

Reply:
Done